builder/assets/clever/CMakeLists.txt

```diff
@@ -0,0 +1,143 @@
+# A CMakeLists file for 'clever' package mirroring 'clover' package
+# (for backwards compatibility)
+
+cmake_minimum_required(VERSION 2.8.3)
+project(clever)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
```

Comment:
I believe this list could be shortened significantly.

Reply:
Yeah, maybe, but anyway this is a hack pseudo package, so it's not intended to be used elsewhere of the clover image. So resolving.

Reply:
Okay, I'll make myself more clear. Please remove everything from that list except `message_generation`, we don't need any other package in this list. The `generate_messages()` call does not need any dependencies as well, since we're only using built-in types (this applies to the "clover" package as well).

A package that calls `find_package(catkin REQUIRED COMPONENTS foo)` and does not list `foo` as its dependency in `package.xml` is malformed, even if it is a "hack pseudo package".